docs/pages/admin-guides/deploy-a-cluster/deploy-a-cluster.mdx

```diff
@@ -6,4 +6,10 @@ description: "Guides to running a self-hosted Teleport cluster in production."
 These guides show you how to run a self-hosted Teleport Enterprise cluster in
 production.
 
+## Dedicated account dashboard
```

Comment:
This is the only heading in this page, which makes it a little harder for me to read:

<img width="749" alt="image" src="https://github.com/user-attachments/assets/a6928748-fcd6-4a75-afb1-41243237ef50">

I would either:

- Remove the heading
- Add another H2, "Guides to self-hosting Teleport", before the `(!toc!)` line.

Reply:
Please see update @ptgott